README.md

```diff
@@ -2,6 +2,9 @@
 
 Credits: João Gaspar, https://github.com/jonnyforestGIS
 
+Last Tested on: QGIS Server 3.22.6 LTR + LizMap Web Client 3.3.6
```

Comment:
FYI 3.3 is not maintained anymore, for a long time already. It was version 3.3.16.

More important when Lizmap Web Client 3.3.6 was released, in May 2020, it was QGIS Server 3.10 the LTR version.
So in the meantime, all QGIS version change some bits of the XML generated (the QGS file), so running LWC 3.3.6 with QGIS Server 3.22, you are running into problems.
Why not upgrading Lizmap Web Client ?

Reply:
Hi, @Gustry thank you for pointed that. 
I notice this situation after the PR. I will update LWC based on is table matrix. 
Cheers.

Reply:
@Gustry, I tested the installation of 3.5.3 but I'm having some issues setting the installation with demo repositories. I add the value

[modules]
lizmap.installparam=demo

in www\lizmap\lizmap\var\config\localconfig.ini.php

and didn't install.

I try as well to run (using the git bash terminal for windows) the lizmap/install/reset.sh --keep-config --demo install but fails in the last lines:

```bash

./reset.sh: line 155: php: command not found
./reset.sh: line 156: php: command not found
./reset.sh: line 165: php: command not found


```

If I look at the code these lines refer to:

```

(line 155) php $SCRIPTDIR/../../lib/jelix-scripts/inifile.php $LIZMAP/var/config/localconfig.ini.php lizmapdemo.path "app:install/lizmapdemo" modules
(line 156) php $SCRIPTDIR/../../lib/jelix-scripts/inifile.php $LIZMAP/var/config/localconfig.ini.php lizmapdemo.access 2 modules
(line 165) (cd $LIZMAP/install && php installer.php)
 ```

Looking at this script, I assume that is trying to put the following lines in the localconfig.ini.php file de values:

```php
[modules]
lizmapdemo.path "app:install/lizmapdemo"
lizmapdemo.access 2
```
this is correct?

Reply:
> and didn't install.

@jonnyforestGIS it is quite some time now that the Lizmap packages do not include anymore the demo projects, see the folder https://github.com/3liz/lizmap-web-client/tree/master/lizmap/install/qgis .In the context of this simplified "package" for Windows we should re-add them. Also we should check/update the readme in order to be sure that all works with osgeo4w v2. Will try to do it in the upcoming days.

Reply:
> Why not upgrading Lizmap Web Client ?

@Gustry mostly lack of time and opportunity, more recently I had to take  leave of absence.

Reply:
> > and didn't install.
> 
> @jonnyforestGIS it is quite some time now that the Lizmap packages do not include anymore the demo projects, see the folder https://github.com/3liz/lizmap-web-client/tree/master/lizmap/install/qgis .In the context of this simplified "package" for Windows we should re-add them. Also we should check/update the readme in order to be sure that all works with osgeo4w v2. Will try to do it in the upcoming days.

I agree @gioman if you want to see some guidelines give me a ping. 

I tested with an osgeo4w v2 but not with Apache Server of the installer. And probably make sense to create a custom or a vhost .conf for lizmap in the osgeo4w v2 installer and who knows add them a new package in the web group (we need to have as well the PHP). This will make it easier the install for users that want to try LWC.

Reply:
No worries @gioman I was just curious and warning you ;-) I hoep you are better after your leave ! (BTW, we will be a few from 3Liz at the FOSS4G/SOTM/QGIS code sprint in Firenze).

Let us know @jonnyforestGIS if we can help on the osgeo4w. It would be great, even if we really recommend using a Linux server.

Reply:
Note that the docker-compose stack has been tested on Windows : https://github.com/3liz/lizmap-docker-compose#windows

Reply:
> No worries @gioman I was just curious and warning you ;-) I hoep you are better after your leave ! (BTW, we will be a few from 3Liz at the FOSS4G/SOTM/QGIS code sprint in Firenze).
> 
> Let us know @jonnyforestGIS if we can help on the osgeo4w. It would be great, even if we really recommend using a Linux server.

Yes, I agree. Using linux server is more powerful (I use) but  I think this kind of configuration (osgeo4w (QGIS Desktop+Server+Lizmap) can be great for Education or training purposes.

It's a good way to facilitate to windows users the use of QGIS Desktop+ QGIS Server and lizmap with a few steps.

For other proposes or advanced users, we recommend the use of Docker or Podman (I'm trying to get some time to test docker-compose in podman 4.1.0 to see if everything runs well).

Reply:
@gustry we update to LCW 3.4.11 thanks to revision of @gioman.
Cheers

Reply:
Nice news !